app/frontend/translations.ts

```diff
@@ -6,6 +6,9 @@ import {Lovelace, CryptoProviderFeature} from './types'
 
 const {ADALITE_MIN_DONATION_VALUE} = ADALITE_CONFIG
 
+const ledgerConnectError =
+  ' Please try connecting you device using the "Connect with WebUSB" functionality (the button underneath "Unlock with Ledger"). For more information please follow the Ledger part of our troubleshooting suggestions.'
```

Comment:
probably redundant space at the beginning and typo - `you` -> `your`

Reply:
Fixed, thnx :+1: